.github/workflows/test.yml

```diff
@@ -56,6 +56,17 @@ jobs:
         --config=clang
         -c opt
         $(bazel query 'kind(was.*_rust_binary, //test/test_data/...)')
+        $(bazel query 'kind(_optimized_wasm_cc_binary, //test/test_data/...)')
+
+    - name: Clean up test data
+      shell: bash
+      run: |
+        # Remove temporal directories
+        for i in $(cd bazel-bin/test/test_data && ls -1); do \
+          if [ -d "bazel-bin/test/test_data/$i" ]; then \
```

Comment:
Consider replacing this with `find bazel-bin/test/test_data/ -mindepth 1 -maxdepth 1 -type d`

Reply:
Thanks! Done!